.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+ [ ] I have searched the [PR](https://github.com/Marketing-Automation-Systems/jellyfish/pulls/) of this repository and believe that this is not a duplicate.
+
+<!-- Checked checkbox should look like this: [x] -->
+
+**Describe the PR**
+
```

Comment:
Maybe give an example here
```suggestion
<!-- For instance: Closes #10  -->
```

Reply:
I don't see purpose to make the example as comment, so I left this phrase on the PR.